Classes/Service/CacheService.php

```diff
@@ -71,6 +80,16 @@ public function flush($force = false)
     }
 
     /**
+     * Flushes caches using the data handler. This should not be necessary any more in the future.
+     * @deprecated
+     * @internal
+     */
+    public function flushCachesWithDataHandler()
+    {
+        $this->dataHandler->clear_cacheCmd('all');
```

Comment:
> No, every action is handled

OK. Then I'm going to assume, that the change here is fine and you don't need any related feature